openapi-cli/src/test/java/io/ballerina/openapi/generators/service/ServiceDiagnosticTests.java

```diff
@@ -128,6 +152,45 @@ public Object[][] singleFileProviderForDiagnosticCheck() {
         };
     }
 
+    @DataProvider(name = "fileProviderForOpenAPI31DiagnosticCheck")
+    public Object[][] fileProviderForOpenAPI31DiagnosticCheck() {
+        return new Object[][]{
+                {"petstore_server_with_base_path.yaml"},
+                {"petstore_get.yaml"},
+                {"openapi_display_annotation.yaml"},
+                {"header_parameter.yaml"},
+                {"petstore_post.yaml"},
+                {"petstore_with_oneOf_response.yaml"},
+                {"response_nested_array.yaml"},
+                {"xml_payload.yaml"},
+                {"xml_payload_with_ref.yaml"},
+                {"duplicated_response.yaml"},
+                {"complex_oneOf_schema.yaml"},
+                {"request_body_ref.yaml"},
+                {"vendor_specific_mime_types.yaml"},
+//                {"ballerinax_connector_tests/ably.yaml"}, // HTTP status code '2XX' is not supported in Ballerina
+                {"ballerinax_connector_tests/azure.iot.yaml"},
+                {"ballerinax_connector_tests/beezup.yaml"},
```

Comment:
Shall we comment on this beezup connector since it has known diagnostic issue https://github.com/ballerina-platform/openapi-tools/issues/1440#issuecomment-1676769160

Reply:
Updated